zerver/lib/events.py

```diff
@@ -502,6 +502,8 @@ def fetch_initial_state_data(
 
     if want("update_display_settings"):
         for prop in UserProfile.property_types:
+            if prop == "enter_sends":
+                continue
```

Comment:
This deserves a comment.

Reply:
I guess it get deleted in the next commit, so maybe it's OK, but I'd still have written this as "# To remove when we make the API change to move this to the `update_display_settings` event type" or something.